cookbooks/swift/recipes/setup.rb

```diff
@@ -239,6 +239,30 @@
   creates "/etc/bash_completion.d/aws_bash_completer"
 end
 
+# rclone setup
+# ~/.config/rclone/rclone.conf
+directory "/home/#{node['username']}/.config" do
+  owner node['username']
+  group node['username']
+  mode 0700
+  action :create
+end
+directory "/home/#{node['username']}/.config/rclone" do
+  owner node['username']
+  group node['username']
+  mode 0700
+  action :create
+end
+template "/home/#{node['username']}/.config/rclone/rclone.conf" do
+  source "/home/rclone/rclone.erb"
+  owner node['username']
+  group node['username']
+  mode 0700
+  variables({
+    :ssl => node['ssl'],
+    :hostname => node['hostname'],
```

Comment:
Might want to use
```
    :base_uri => node['base_uri'],
    :auth_uri => node['auth_uri'],
```
instead, just to minimize the places where we have to know how to reconstruct them.

Reply:
ahh, yeah, much better!
Mental note, check all the vars before trying to build your own ;)